crates/bevy_text/src/glyph_brush.rs

```diff
@@ -209,3 +200,30 @@ impl GlyphPlacementAdjuster {
         Vec2::new(self.0, 0.) + v
     }
 }
+
+pub fn compute_text_bounds<'a, T>(
```

Comment:
This either needs to be `pub(crate)` or documented explaining when and how to be used

Reply:
`pub(crate)` I think.

It's probably not that useful to users and potentially confusing. I can imagine someone struggling to use it to compute the size of a text block, instead of just retrieving the size from TextLayoutInfo.